src/ucp/rndv/rndv_get.c

```diff
@@ -30,7 +30,7 @@ ucp_proto_rndv_get_common_init(const ucp_proto_init_params_t *init_params,
     ucp_context_t *context               = init_params->worker->context;
     ucp_proto_multi_init_params_t params = {
         .super.super         = *init_params,
-        .super.cfg_thresh    = ucp_proto_rndv_cfg_thresh(context, rndv_modes),
+        .super.cfg_thresh    = UCS_MEMUNITS_AUTO,
```

Comment:
Q: with older implementation, the resulting threshold could be 0 or AUTO, while with the new one it is always AUTO.
Is this change on purpose? May it introduce regressions?

Reply:
This change is why this patch was created. In master branch if RNDV_SHCEME is set to certain RNDV mode, and this protocol is unavailable, UCX falls back to other RNDV.

It was decide to remove this fall back logic and just don't initialize RNDV modes which are not set in RNDV_SCHEME.